lfs/gitscanner_catfilebatchcheckscanner_test.go

```diff
@@ -0,0 +1,34 @@
+package lfs
+
+import (
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCatFileBatchCheckScanner(t *testing.T) {
```

Comment:
Alternatively could be broken down into smaller tests, i.e., one for "short line", one for invalid sizes, one for long lines, etc.

At that point I'd probably extract a test helper function or type, but I think that'd be a cool improvement in that it'd allow you to name the tests more specifically.